test/controllers/iterations_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class IterationsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I would like to see tests that verify this controller can perform CRUD actions through JSON endpoints.

Reply:
Hey, I don't have access to team-automation-calculator. I can't push or create specified branch team-automation-calculator so that I have to fork it for creating the pull request.